src/interfaces/assistants_web/src/hooks/use-files.ts

```diff
@@ -12,6 +14,29 @@ import { useConversationStore, useFilesStore, useParamsStore } from '@/stores';
 import { UploadingFile } from '@/stores/slices/filesSlice';
 import { fileSizeToBytes, formatFileSize, getFileExtension, mapExtensionToMimeType } from '@/utils';
 
+export const useFile = ({
+  fileId,
+  agentId,
+  conversationId,
+}: {
+  fileId: string;
+  agentId?: string;
+  conversationId?: string;
+}): UseQueryResult<AgentFileFull | ConversationFileFull, Error> => {
```

Comment:
Actually, looking at the upstream python code, it may make sense to just have one type for uploaded files returned by the API, regardless of whether they come from the `conversation/get_file` or `agent/get_agent_file` routes. Is the distinction between the two necessary?

Reply:
I agree, let's create a common class